components/input/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="inputstyle.css">
+</head>
+<body>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    
+    
+    <div class="input-box">
+        <input class="input1" placeholder="Email Id"></input>
```

Comment:
`input1`, `input2`, `input3` are not every meaning full. You can try with a base class name that will provide basic styling then you can add other classes on top of it to change the styling for example in case of error. you can try some thing like `.form-control .form-control-invalide`

Reply:
thanks this one is resolved